npm/qsharp/ux/circuit-vis/panel.ts

```diff
@@ -268,60 +270,71 @@ const _gate = (
  * Interface for gate dictionary
  */
 interface GateDictionary {
-  [index: string]: Operation;
+  [index: string]: Component;
 }
 
 /**
  * Object for default gate dictionary
  */
 const defaultGateDictionary: GateDictionary = {
```

Comment:
This whole declaration was kind of redundant to begin with (all the `targets` fields are the same) so I don't find the addition of another field that offensive

Reply:
I've written a helper function to make that dictionary definition a little more readable.